src/server/services/bounty.service.ts

```diff
@@ -66,7 +67,15 @@ export const getAllBounties = <TSelect extends Prisma.BountySelect>({
     });
   }
 
-  const orderBy: Prisma.BountyFindManyArgs['orderBy'] = [{ complete: 'asc' }];
+  if (status) {
+    if (status === BountyStatus.Open) AND.push({ complete: false, expiresAt: { gt: new Date() } });
+    else if (status === BountyStatus.Awarded) AND.push({ complete: true, entries: { some: {} } });
```

Comment:
This is interesting, `some: {}` works? I'd assume you need to provide something here 🤔 The more you know 👍